network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/NetworkImpl.java

```diff
@@ -652,6 +653,36 @@ public void removeListener(NetworkListener listener) {
         return listeners;
     }
 
+    @Override
+    public <C extends Connectable> Iterable<C> getConnectables(Class<C> clazz) {
+        return getConnectableStream(clazz).collect(Collectors.toList());
+    }
+
+    @Override
+    public <C extends Connectable> Stream<C> getConnectableStream(Class<C> clazz) {
+        return index.getIdentifiables().stream().filter(clazz::isInstance).map(clazz::cast);
```

Comment:
I think `LccConverterStations` are missing and maybe other things ?

Reply:
Added. To me, they are the only ones missing